Surface_mesher/doc/Surface_mesher/PackageDescription.txt

```diff
@@ -38,7 +38,7 @@
 \cgalPkgDependsOn{\ref PkgTriangulation3}
 \cgalPkgBib{cgal:ry-smg}
 \cgalPkgLicense{\ref licensesGPL}
-\cgalPkgDemo{Surface Mesh Generator,surface_mesher.zip}
+\cgalPkgDemo{Polyhedron demo,polyhedron_3.zip}
```

Comment:
```suggestion
```

The polyhedron demo does not use `Surface_mesher` at all (grepped for it). It uses Mesh_3 to implement similar functions.